saasherder/saasherder.py

```diff
@@ -129,14 +130,39 @@ def apply_filter(self, template_filter, data):
 
         return yaml.dump(data_obj, encoding='utf-8', default_flow_style=False)
 
-    def apply_saasherder_annotations(self, data, service_name):
+    def apply_saasherder_labels_and_annotations(self, data, service_name, saas_repo_url):
```

Comment:
I don't like the naming of this method. It's confusing, and why does it return two values?